index.html

```diff
@@ -74,6 +75,34 @@ <h2 class="wotd"></h2>
     </div>
   </div>
 
+   <div class="definition-container hidden" data-view="definition-view">
+     <div class="gray-background">
+       <div class="definition-row">
+         <h2 class="wotd-defintion"></h2>
+         <h3 class="partOfSpeech"></h3>
+         <div class="definition-content">
+            <h3>definition</h3>
+            <p class="definition">n/a</p>
+         </div>
+         <div class="definition-content">
+            <h3>example</h3>
+            <p class="example">n/a</p>
```

Comment:
You should change the text content from "n/a" to "Loading..." so it doesn't look like you have no data